### PR TITLE
Portfolio build

### PR DESCRIPTION
**Pull request by robertkieronreeves**
Browser tested.

I have taken inspiration from Interneting is hard. More specifically the web typography page.

[https://internetingishard.com/html-and-css/web-typography/](url).

As far as content goes, this is still in progress but the basic layout and template are complete and bug free.

### CODE REVIEW

File: Portfolio.html [hunk 1]
```diff
@@ -4,11 +4,76 @@
...
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="page-header">
+            <div class="main-header-wrapper">
+                <div class="main-header-centre">
```
Comment (Benson-J) [inline Portfolio.html]: Necessary?  Doesn't seem to have CSS.

File: Portfolio.html [hunk 2]
```diff
@@ -4,11 +4,76 @@
...
+    <nav class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="page-header">
```
Comment (Benson-J) [inline Portfolio.html]: Consider using semantic header tag rather than class

File: Portfolio.html [hunk 3]
```diff
@@ -4,11 +4,76 @@
...
+            <h2>------ ABOUT ME ------</h2>
+            <p class="main-content-text">
+                Lots of interesting stuff about me, all of which is indeed very interesting and it will be so interesting that you will not fall asleep reading it all I promise. Are you not entertained yet?
+            </p>
+            <p class="main-content-wrapper">Lots more interesting stuff I'm sure you will be delighted to see.
+            </p>
+                <div class="blog-container" id="C2">
+                <h2>----- THREE PEDALS ONLY -----</h2>
+                    <p class="main-content-text">
+                Lots of interesting stuff here about manual gearboxes and how manual gearboxes are dying out and why this is such a bad thing for car enthusiasts everywhere and it's better for a driver's car to have a manual than an auto.
```
Comment (Benson-J) [inline Portfolio.html]: Add tabs to align properly

File: Portfolio.html [hunk 4]
```diff
@@ -4,11 +4,76 @@
...
+                        <p class="sub-heading2">There are lots of interesting things about me on this website</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    <div class="main-content" id="C1">
+        <div class="main-content-wrapper">
+            <h2>------ ABOUT ME ------</h2>
+            <p class="main-content-text">
+                Lots of interesting stuff about me, all of which is indeed very interesting and it will be so interesting that you will not fall asleep reading it all I promise. Are you not entertained yet?
```
Comment (Benson-J) [inline Portfolio.html]: Seems to be some p tags with content on same line, & some with content on it's own line.  Might be better to be consistent?

File: Portfolio.html [hunk 5]
```diff
@@ -4,11 +4,76 @@
...
+                        <img class="gearbox" src="https://user-images.githubusercontent.com/271652/38042847-6d825426-3283-11e8-8b37-bb58514c3a2f.png" alt="stick shift">
+                        <p class="blog-text-paragraph">Stuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxesStuff here about manual gearboxes</p>
+                    </div>
+                </div>
+        </div>
+    </div>
 
+    <div class="footer" id="C3">
```
Comment (Benson-J) [inline Portfolio.html]: Consider using semantic footer tag rather than class & ID

File: Portfolio.css [hunk 6]
```diff
@@ -1,3 +1,126 @@
...
+
 Body {
     font-family: 'Quicksand', sans-serif;
+    font-size: 62.5%;
+    background-color: #fff;
+    color: #5D6063;
+}
+
+.main-nav {
+    background-color: #ffffff;
```
Comment (Benson-J) [inline Portfolio.css]: Inconsistent use between #fff & #ffffff

File: Portfolio.css [hunk 7]
```diff
@@ -1,3 +1,126 @@
...
+    bottom: 5px;
+}
+
+.nav-text {
+    margin-left: 15px;
+}
+
+.main-header-wrapper {
+    max-width: 580px;
+    margin-left: auto;
```
Comment (Benson-J) [inline Portfolio.css]: Setting padding/margin for only top & bottom, or left & right, can be done on 1 line.

**Review by Benson-J — COMMENTED** [review]
Good, a few possible changes listed.

**Review by Benson-J — COMMENTED** [review]
Good, a few possible changes listed.

File: Portfolio.html (hunk 2 above)
Comment (robertkieronreeves) [reply Portfolio.html]: Done.

File: Portfolio.css (hunk 6 above)
Comment (robertkieronreeves) [reply Portfolio.css]: Done

File: Portfolio.html (hunk 3 above)
Comment (robertkieronreeves) [reply Portfolio.html]: done

File: Portfolio.html (hunk 4 above)
Comment (robertkieronreeves) [reply Portfolio.html]: done.

**Pull request closed without merging**